### PR TITLE
Add Ralph Merkle's PhD Thesis

### PR DESCRIPTION
**Pull request by tombusby**
In this document he proposes the hash function so it's pretty notable.

Not sure if I got the format of the JSON object right, so let me know and I'll fix it.

### CODE REVIEW

**Comment by tombusby** [comment]
I've just noticed your PR where you convert all this to YAML.

So, let's leave this open for now so I don't forget about it, but when you've merged your PR, I'll resubmit.

File: data/literature.json [hunk 1]
```diff
@@ -872,5 +872,17 @@
         "categories": ["economics", "cryptography", "law"],
         "doctype": "essay",
         "external": "http://web.archive.org/web/20140406003401/szabo.best.vwh.net/idea.html"
-    }]
+    },
+    {
+        "id": 85,
+        "title": "Secrecy Authentication and Public Key Systems",
```
Comment (bitstein) [inline data/literature.json]: Please fix this to `"Secrecy, Authentication, and Public Key Systems"`

File: data/literature.json [hunk 2]
```diff
@@ -872,5 +872,17 @@
...
+        "author": ["ralph-merkle"],
+        "date": "1979",
```
Comment (bitstein) [inline data/literature.json]: Please fix this to `"June 1979"`

File: data/literature.json [hunk 3]
```diff
@@ -872,5 +872,17 @@
...
+        "slug": "secrecy-authentication-and-public-key-systems",
+        "formats": ["ext"],
```
Comment (bitstein) [inline data/literature.json]: Please fix this to `["pdf", "ext"]`

**Comment by tombusby** [comment]
Fixed

**Comment by bitstein** [comment]
Thanks!

**Pull request merged**